[BUG] credential migration error in log doing weblogin since 2.0

### Title

Failed to migrate credential

### Platform

Website - Safari

### Server Installation Method

Docker

### Version

2.0.0

### Troubleshooting

- [x] I have examined logs and tried to find the issue
- [x] I have reviewed opened and closed issues
- [x] I have tried restarting the application
- [x] I have read Known Issues (https://github.com/Termix-SSH/Support/issues/296)

### The Problem

I found this in the los, the login works but I think this is not wanted. The user has 2FA activated.
In the log output I have delete the user:hash-id an replace it with „xxxx“

```
termix  | [9:19:59 AM] [ERROR] [🗄️] Failed to migrate credential [user:xxxxxxxxxx_xx]
termix  | Error: Unsupported state or unable to authenticate data
termix  |     at Decipheriv.final (node:internal/crypto/cipher:184:29)
termix  |     at FieldCrypto.decryptField (file:///app/dist/backend/backend/utils/field-crypto.js:66:31)
termix  |     at CredentialSystemEncryptionMigration.migrateUserCredentials (file:///app/dist/backend/backend/utils/credential-system-encryption-migration.js:33:39)
termix  |     at async AuthManager.performLazyEncryptionMigration (file:///app/dist/backend/backend/utils/auth-manager.js:85:36)
termix  |     at async AuthManager.authenticateUser (file:///app/dist/backend/backend/utils/auth-manager.js:59:13)
termix  |     at async file:///app/dist/backend/backend/database/routes/users.js:1232:28
```

### How to Reproduce

1. Update zu 2.0.0
2. login with my existing user
3. docker compose logs

### Additional Context

_No response_

--- a/Casks/termix.rb
+++ b/Casks/termix.rb
@@ -1,6 +1,6 @@
 cask "termix" do
   version "2.0.0"
-  sha256 "a752ad4f05b4991b8a2ef986da80a86b79a70c93fe1045c4399fcf348a28c1d0"
+  sha256 "2e381ac504093bfa72d16ee20043640724394729b2ca0e6b26c9eac19aeb6d08"
 
   url "https://github.com/Termix-SSH/Termix/releases/download/release-#{version}-tag/termix_macos_universal_dmg.dmg"
   name "Termix"

--- a/src/backend/utils/credential-system-encryption-migration.ts
+++ b/src/backend/utils/credential-system-encryption-migration.ts
@@ -107,20 +107,23 @@ export class CredentialSystemEncryptionMigration {
 
           migrated++;
         } catch (error) {
-          databaseLogger.error("Failed to migrate credential", error, {
-            credentialId: cred.id,
-            userId,
-          });
+          databaseLogger.warn(
+            `Skipping credential migration for credential ${cred.id}: ${error instanceof Error ? error.message : "Unknown error"}`,
+            {
+              operation: "credential_migration_skip",
+              credentialId: cred.id,
+              userId,
+            },
+          );
           failed++;
         }
       }
       return { migrated, failed, skipped };
     } catch (error) {
-      databaseLogger.error(
-        "Credential system encryption migration failed",
-        error,
+      databaseLogger.warn(
+        "Credential system encryption migration incomplete",
         {
-          operation: "credential_migration_failed",
+          operation: "credential_migration_incomplete",
           userId,
           error: error instanceof Error ? error.message : "Unknown error",
         },

